### ISSUE TITLE
no-unused-variable, should be split into no-unused-variable and no-unused-import

### ISSUE THREAD

**Issue by maraisr** [palantir/tslint#3317]
### Bug Report

- __TSLint version__: 5.7.0
- __TypeScript version__: next
- __Running TSLint via__:  yarn scripts

#### TypeScript code being linted

```ts
import {xyz} from 'abc';
import {foo} from 'bar';

class MyPage {
    test: string = xyz();
}
```

with `tslint.json` configuration:

```json
{
  "rules": {
    "no-unused-variable": true
  }
}
```

#### Actual behaviour / Expected.
With fix mode it strips the "test" property because its not being used. But it actually is (via a template that this class gets bind too).

Is there a way to just have unused imports checked, but leave my variables alone?

**Comment by ajafff** [comment]
`no-unused-variable` only complains about unused private properties. Your property `test` is public. That means your code snippet must be wrong.

**Comment by JoshuaKGoldberg** [comment]
`no-unused-variable` now uses diagnostics directly from TypeScript. Closing this issue as we don't want to maintain _two_ rules that mirror TypeScript's diagnostics.
